src/components/Layout/components/Header/header.styles.ts

```diff
@@ -34,7 +34,7 @@ export const Left = styled.div`
   justify-content: flex-start;
 
   .MuiButton-navPrimary {
-    &:first-of-type {
+    &:first-child {
```

Comment:
@frano-m, can you double-check this for me? I was getting a left margin on the "Metadata" menu button as it is the first button element in the nav and have limited the left margin to just the first child.

Reply:
Good pick up!

Using `first-child` will render a warning in the console:

```
The pseudo class ":first-child" is potentially unsafe when doing server-side rendering. Try changing it to ":first-of-type". Error Component Stack
```

See [MarkdownRenderer](https://github.com/DataBiosphere/findable-ui/blob/main/src/components/MarkdownRenderer/markdownRenderer.styles.ts#L5-L11) component for a workaround on the warning. Include `:not(style)` as well.